client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/style.scss

```diff
@@ -77,6 +77,11 @@
 	.gridicon.is-warning {
 		color: var( --color-warning );
 	}
+	.gridicon.label-purchase-modal__step-spinner {
+		fill: var( --color-neutral-dark );
```

Comment:
WDYT about preserving the spinner's pink color here?

<img width="668" alt="image" src="https://user-images.githubusercontent.com/1759681/203588984-60d450bf-23ec-4b11-acc8-cf83311ef59c.png">

```suggestion
		fill: var( --color-accent );
```

As much as I like the decalypsoification in this PR and your code, I am wondering if maybe we should copy the original spinner from Calypso here. 🤔 It is a very small component and I think it looks better (IMHO!)

What do you think?

Reply:
+1 on the Calypso spinner. I will clone it over.